### PR TITLE
Design tokens as source of truth for typography

### PR DESCRIPTION
**Pull request by jamesmockett**
## What are you changing?

Updates Source Foundations to use `@csnx/design-tokens` as the source of truth for typography tokens.

### CODE REVIEW

File: libs/@guardian/source-foundations/src/space/space.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { tokens } from '@csnx/design-tokens';
-import { pxStringToNumber, pxToRem } from '../utils/px-to-rem';
```
Comment (oliverabrahams) [inline libs/@guardian/source-foundations/src/space/space.ts:2]: Nice

File: libs/@guardian/source-foundations/src/utils/convert-value.test.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			'Arial',
+			'Lucida Grande',
+			'sans-serif',
+		];
+		const result = fontArrayToString(value);
+		expect(result).toBe(
+			'GuardianTextSans, "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif',
+		);
+	});
+});
```
Comment (oliverabrahams) [inline libs/@guardian/source-foundations/src/utils/convert-value.test.ts:46]: Good idea

**Review by oliverabrahams — APPROVED** [review]
Looks good

**Review by oliverabrahams — APPROVED** [review]

File: libs/@csnx/design-tokens/tokens.json [hunk 3]
```diff
@@ -770,185 +770,203 @@
...
-	},
-	"titlepiece": {
-		"$type": "typography",
-		"small": {
-			"$value": {
-				"fontFamily": "{font.family.titlepiece}",
-				"fontSize": "{font.size.42}",
-				"lineHeight": "{font.lineHeight.tight}",
-				"fontWeight": "{font.weight.bold}"
+		"textDecorationThickness": {
```
[the remark this answers is not in the document]
Comment (jamesmockett) [reply libs/@csnx/design-tokens/tokens.json:774]: Ooo, good spot! You're right, it should. The unitless values work here as `px` gets appended when they're used elsewhere in the typography styles, but we should be consistent so the tokens make sense when we output them as different formats.

[the remark this answers is not in the document]
Comment (jamesmockett) [reply libs/@csnx/design-tokens/tokens.json:774]: Done. The CSS output makes more sense now as you get this:
`--source-typography-textDecorationThickness-4: 0.25rem;`

**Pull request merged**